src/WinGet.RestSource.IntegrationTest/TestCollateral/Manifests/powertoys.json

```diff
@@ -0,0 +1,91 @@
+﻿{
+  "PackageIdentifier": "Microsoft.PowerToys",
```

Comment:
>Microsoft.PowerToys [](http://example.com/codeflow?start=24&length=19)

You might want to consider moving the package identifier and software name to a different location "RestIntegrationTest.PowerToys" perhaps. Otherwise we may run into a collision with the other components (service publish).

Reply:
Fixed, thanks.